### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rinigus**
Here is a README describing main policies and how to use Chum repos. Feel free to correct, change as needed. 

As it is through the branch in this repository, please change in the branch directly.

Fixes: #1

### CODE REVIEW

**Comment by rinigus** [comment]
@piggz: I have added also packaging script to this PR. Will test it now at OBS as well.

**Comment by rinigus** [comment]
Added a detail regarding submitted software version and running `ssu ur` on install of the package.

@piggz, please review.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,161 @@
...
+
+
+## How to use these repositories: users
+
+At the moment of writing, there is no GUI for selecting packages for
+installation and it has to be done through command line tools
+(`zypper` or `pkcon`).
+
+To use the repositories on your device, users would have to
+- install package `sailfishos-chum`;
```
Comment (piggz) [inline README.md:38]: Whats this?

Comment (rinigus) [reply README.md:38]: it is a small SPEC installing SSU ini file pointing to chum repository. SPEC and INI are included in this PR as well.

Compiled at https://build.merproject.org/package/show/sailfishos:chum/sailfishos-chum and can be distributed at OpenRepos as well. Just to make it easy to add chum repo for users in a way that will be pointing to the correct SFOS version repository

File: README.md [hunk 2]
```diff
@@ -1,7 +1,161 @@
...
+upstream as submodule and packaging scripts/patches into the packaging
+repository.
+
+
+## Q&A
+
+- My software requires specific SFOS versions, can I still use Chum
+  repositories? Yes, you can. You would just have to disable the
+  versions and architectures of SFOS that are not supported in OBS
+  package Meta settings. Example: see Pure Maps.
```
Comment (piggz) [inline README.md:147]: Should we mention the macros?

File: README.md (hunk 1 above)
Comment (piggz) [reply README.md:38]: Ah, its below!

File: README.md (hunk 2 above)
Comment (rinigus) [reply README.md:147]: Yes, we should. Although they are so far defined for 4.1.0.24 only, if I remember correctly. something we will have to ask lbt to fix.

Comment (piggz) [reply README.md:147]: No, theyre here too https://build.sailfishos.org/project/prjconf/sailfishos:4.0.1.48

Comment (rinigus) [reply README.md:147]: Not in https://build.sailfishos.org/project/prjconf/sailfishos:3.4.0.24 3.3.0.16 3.2.1.20

Comment (rinigus) [reply README.md:147]: For clarity: macros are a separate question. Namely, if you need to adjust something in SPEC then they will help. Here, case is when you cannot get it working for earlier versions at all. For Pure Maps, question is in gcc version required by Mapbox GL QML plugin.

Do you want to write Q&A for macros or should I fill it? I can probably do it in ~2 hours

**Comment by rinigus** [comment]
Thanks for merging! Just added SFOS version macro description . Out-of-sync :)

**Pull request merged**